### PR TITLE
WD RT JUN 2022 | Ruben Andino

### PR DESCRIPTION
**Pull request by rubenanlo**
Finished all iterations minus bonuses. Added a very simple format :)

thanks for reviewing!!

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 // 👇 Start handling routes here
 const index = require('./routes/index');
 app.use('/', index);
+const celebrities = require('./routes/celebrities.routes');
+app.use('/', celebrities);
+const movies = require('./routes/movies.routes');
+app.use('/', movies);
```
Comment (TA-Remote) [inline app.js:33]: for shorter syntax you can write it like this:

```
app.use('/', require('./routes/celebrities.routes'));
```

File: routes/celebrities.routes.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    .then((celebritiesDetails) => {
+        const data = {celebritiesDetails}
+        res.render('celebrities/new-celebrity',data)
+    })
+    .catch((error) => {
+        console.log("Error retrieving celebrities from DB, ", error)
+        next(error);
+    })
+
+});
```
Comment (TA-Remote) [inline routes/celebrities.routes.js:30]: In the GET route where you have form to create some celebrity -> there is no need to access database! All you need to do is render the hbs with form ;)

```
router.get("/celebrities/create", (req, res) => {
     res.render('celebrities/new-celebrity')
});
```

**Review by TA-Remote — COMMENTED** [review]
Excellent as always Ruben! 👌 Keep it up 💪

File: app.js (hunk 1 above)
Comment (rubenanlo) [reply app.js:33]: thanks! That looks much better

**Pull request closed without merging**